app/src/main/res/values/themes.xml

```diff
@@ -1,7 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-
     <style name="Theme.Oipie" parent="android:Theme.Material.Light.NoActionBar">
         <item name="android:statusBarColor">@color/purple_700</item>
+        <item name="android:navigationBarColor">@android:color/transparent</item>
+    </style>
+    <style name="Theme.Oipie.Starting" parent="Theme.SplashScreen">
```

Comment:
Could we put this in a separate file? To allow light/dark mode easily.
```
<resources>
 <style name="Theme.Oipie.Starting" parent="Theme.SplashScreen">
       <item name="windowSplashScreenBackground">@color/purple_700</item>
        <item name="windowSplashScreenAnimatedIcon">@mipmap/ic_broccoli_foreground</item>
        <item name="windowSplashScreenAnimationDuration">500</item>
        <item name="postSplashScreenTheme">@style/Theme.Oipie</item>
    </style>
</resources>
```

Reply:
Yes, i will fix it and push the changes.